backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/migrations/V0028_Add_Application_Status.kt

```diff
@@ -0,0 +1,23 @@
+package app.ehrenamtskarte.backend.migration.migrations
+
+import app.ehrenamtskarte.backend.migration.Migration
+import app.ehrenamtskarte.backend.migration.Statement
+
+@Suppress("ClassName")
+internal class V0028_Add_Application_Status : Migration() {
+    override val migrate: Statement = {
+        exec(
+            """
+            -- Add column "status"
+            alter table "applications" add column "status" varchar(32) not null default 'Pending'
+                 check ("status" in ('Pending', 'Approved', 'ApprovedCardCreated', 'Rejected'));
+            -- Set "status" to 'ApprovedCardCreated' for rows that have "cardCreated" set to true
+            update "applications" set "status" = 'ApprovedCardCreated' where "cardCreated";
+            -- Remove ”cardCreated" column
+            alter table "applications" drop column "cardCreated";
+            -- Add column "statusResolvedDate"
```

Comment:
🙃 personal preference, feel free to ignore:
Smthg like statusUpdatedAt sounds more precise to me, considering that it can be updated more than once

Reply:
> The field captures the time an application changes from Pending to either Approved or Rejected

this field is also changed when the card is issued. is it intended?

Reply:
Ah the condition guarding the change to this field was wrong, I updated it, should work now as intended.